webhooks-extension/pkg/endpoints/webhook.go

```diff
@@ -425,9 +427,9 @@ func (r Resource) createWebhook(request *restful.Request, response *restful.Resp
 		}
 		_, varexists := os.LookupEnv("PLATFORM")
 		if !varexists {
-			ingressTaskRun, err := r.createIngressTaskRun("create", installNs)
```

Comment:
Hi @akihikokuroda thank you for the PR. Please could you ensure that the createIngressTaskRun method is deleted, along with its Task definitions and any other related resources. Thank you!

Reply:
Thanks!  I removed the go function and the task in 2 yaml files.